tests/parsers/thermo_fisher_qubit_flex/testdata/thermo_fisher_qubit_flex_example_01.json

```diff
@@ -13,25 +13,25 @@
                                     {
                                         "device type": "fluorescence detector",
                                         "custom information document": {
-                                            "sample volume setting": {
-                                                "value": 1,
-                                                "unit": "μL"
-                                            },
                                             "operating minimum": {
                                                 "value": "10.0-20.0",
                                                 "unit": "ng/µL"
                                             },
+                                            "operating maximum": {
```

Comment:
NOTE to reviewer - the only changes in test data are random reorderings, and changing some ints to floats (that should be floats)